app/assets/javascripts/discourse/controllers/tag-groups-new.js.es6

```diff
@@ -0,0 +1,14 @@
+import Controller from "@ember/controller";
+
+export default Controller.extend({
+  tagGroups: Ember.inject.controller(),
+
+  actions: {
+    onSave() {
+      let tagGroups = this.tagGroups.model;
```

Comment:
```suggestion
      const tagGroups = this.tagGroups.model;
```

Reply:
a `pushObject` is not a mutation of the object

Reply:
I'm of a Yehuda's school of thought here (https://twitter.com/wycats/status/1178165303689105410), i.e. prefer `let` everywhere except the actual, immutable, top-level constants. 🙂

But I'm committing your changes for now, we can have a bikeshedding discussion later. 😜